src/tallies/filter.cpp

```diff
@@ -151,6 +152,8 @@ Filter* Filter::create(const std::string& type, int32_t id)
     return Filter::create<SpatialLegendreFilter>(id);
   } else if (type == "sphericalharmonics") {
     return Filter::create<SphericalHarmonicsFilter>(id);
+  } else if (type == "time") {
```

Comment:
Perhaps one can refactor the code (later) and make the strings static variables of the classes. 
If there is only one definition of the actual string, no typos can occure.

Reply:
Good point -- we do have a `type()` member function which gives that string, but it requires that you already have an instance. With a little refactoring this could be cleaned up, but let's leave that for another time if it's alright with you.